### PR TITLE
itemWidth for MazSelect

### PR DESCRIPTION
**Pull request by vedatyilmaz**
add itemWidth option for MazSelect

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by LouisMazel** [comment]
Hi @vedatyilmaz,

The prop `max-list-width` is not enough for your use case ?

**Comment by vedatyilmaz** [comment]
Hi @LouisMazel 
it could be useful min-list-width to specify minimum width and item-width to specify exact width for items

**Comment by LouisMazel** [comment]
> Hi @LouisMazel it could be useful min-list-width to specify minimum width and item-width to specify exact width for items

Ok, so I will review your MR. I change the target branch of the PR to a release branch

File: packages/lib/package/components/MazSelect.vue [hunk 1]
```diff
@@ -130,6 +130,7 @@
       },
     },
     itemHeight: { type: Number, default: 40 },
+    itemWidth: { type: Number, default: 140 },
```
Comment (LouisMazel) [inline packages/lib/package/components/MazSelect.vue:133]:
```suggestion
    itemWidth: { type: Number, default: undefined },
```

Don't set a default value to not change the initial behavior, by default the width of the item/option should be 100% of the parent (list)

File: packages/lib/package/components/MazSelect.vue [hunk 2]
```diff
@@ -57,7 +57,7 @@
             },
             `--${color}`,
           ]"
-          :style="{ height: `${itemHeight}px` }"
+          :style="{ height: `${itemHeight}px`, width: `${itemWidth}px` }"
```
Comment (LouisMazel) [inline packages/lib/package/components/MazSelect.vue:60]: Use a computed to apply the style of this element

```suggestion
          :style="itemsStyle"
```

And something like that

```ts
const itemsStyle = computed(() => {
  const width = typeof props.itemWidth === 'number' ? `${props.itemWidth}px` : undefined
  
  return { height: `${props.itemHeight}px`, width }
})
```

**Review by LouisMazel — CHANGES_REQUESTED** [review]

**Pull request closed without merging**